<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1200</width>
                <height>800</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>2D 看图与标注 - Qt + OpenCV</string>
        </property>
        <widget class="QWidget" name="centralwidget"/>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>1200</width>
                    <height>22</height>
                </rect>
            </property>
            <widget class="QMenu" name="menuFile">
                <property name="title">
                    <string>文件(&amp;F)</string>
                </property>
                <addaction name="actionOpen"/>
                <addaction name="actionSaveAs"/>
            </widget>
            <widget class="QMenu" name="menuTools">
                <property name="title">
                    <string>工具(&amp;T)</string>
                </property>
                <addaction name="actionSelect"/>
                <addaction name="actionRect"/>
                <addaction name="actionCircle"/>
                <addaction name="actionClear"/>
            </widget>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
        <widget class="QToolBar" name="toolBar">
            <property name="windowTitle">
                <string>工具</string>
            </property>
            <attribute name="toolBarArea">
                <enum>TopToolBarArea</enum>
            </attribute>
            <attribute name="toolBarBreak">
                <bool>false</bool>
            </attribute>
            <addaction name="actionOpen"/>
            <addaction name="actionSaveAs"/>
            <addaction name="separator"/>
            <addaction name="actionSelect"/>
            <addaction name="actionRect"/>
            <addaction name="actionCircle"/>
            <addaction name="separator"/>
            <addaction name="actionClear"/>
        </widget>
        <action name="actionOpen">
            <property name="text">
                <string>打开...</string>
            </property>
            <property name="toolTip">
                <string>打开图像</string>
            </property>
        </action>
        <action name="actionSaveAs">
            <property name="text">
                <string>另存为...</string>
            </property>
            <property name="toolTip">
                <string>保存合成图像</string>
            </property>
        </action>
        <action name="actionSelect">
            <property name="checkable">
                <bool>true</bool>
            </property>
            <property name="text">
                <string>选择</string>
            </property>
            <property name="toolTip">
                <string>选择/浏览</string>
            </property>
        </action>
        <action name="actionRect">
            <property name="checkable">
                <bool>true</bool>
            </property>
            <property name="text">
                <string>画矩形</string>
            </property>
            <property name="toolTip">
                <string>绘制矩形框</string>
            </property>
        </action>
        <action name="actionCircle">
            <property name="checkable">
                <bool>true</bool>
            </property>
            <property name="text">
                <string>画圆</string>
            </property>
            <property name="toolTip">
                <string>绘制圆形</string>
            </property>
        </action>
        <action name="actionClear">
            <property name="text">
                <string>清除标注</string>
            </property>
            <property name="toolTip">
                <string>清空全部矩形与圆</string>
            </property>
        </action>
    </widget>
    <resources/>
    <connections/>
</ui>